sqlframe/databricks/dataframe.py

```diff
@@ -53,7 +53,7 @@ def _typed_columns(self) -> t.List[CatalogColumn]:
                 CatalogColumn(
                     name=normalize_string(
                         row.col_name, from_dialect="execution", to_dialect="output"
-                    ),
+                    ).replace("`", ""),
```

Comment:
Can you provide some context in terms of why this is needed? It looks like `normalize_string` is quoting the values but then you are removing them after.

Reply:
Hi,
This is not really needed, I've reverted this change